sdk/go/common/workspace/plugins.go

```diff
@@ -894,7 +969,7 @@ func NewPluginSpec(
 				return PluginSpec{}, errors.New("cannot specify a plugin download URL when the plugin name is a URL")
 			}
 			name = strings.ReplaceAll(u.RequestURI(), "/", "_")
-			pluginDownloadURL = u.String()
+			pluginDownloadURL = "git://" + u.String()
```

Comment:
Again, I don't understand why we append `git://` here

Reply:
Yea maybe worth a comment to just highlight why this is correct/unconditionally safe?